pom.xml

```diff
@@ -799,17 +799,39 @@ under the License.</licenseText>
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-pmd-plugin</artifactId>
```

Comment:
its already there. It takes 5 seconds so might be justifiable to run for local build.